dfm-sideline-sidekick-app/App.tsx

```diff
@@ -1,13 +1,31 @@
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
 import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
+import { StyleSheet } from "react-native";
+
+import ArrayPage from "./components/ArrayPage";
+import Home from "./pages/Home";
+import GeneralPrinciples from "./pages/generalPrinciples";
+const Stack = createStackNavigator();
 
 export default function App() {
   return (
-    // eslint-disable-next-line @typescript-eslint/no-use-before-define
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen name="Home" component={Home} />
```

Comment:
first fix is to pull from master because app.tsx is a merge conflict but that should be pretty quick. In terms of the stack screen, I believe we should add the options={{ headerShown: false }} here? Especially since the navbar is not synced with history of screens visited, imo its just best to not show the headers. Final call on this is up to @Anthonyp0329 though